test/XRP/default-xrp-client.test.ts

```diff
@@ -669,15 +669,15 @@ describe('Default XRP Client', function (): void {
     const transactionHistory = await xrpClient.paymentHistory(testAddress)
 
     // THEN the returned transactions are conversions of the inputs with non-payment transactions filtered.
-    const expectedTransactionHistory: Array<XRPTransaction> = XRPTestUtils.transactionHistoryToPaymentsList(
+    const expectedTransactionHistory: Array<XrpTransaction> = XRPTestUtils.transactionHistoryToPaymentsList(
```

Comment:
Can you add a TODO to rename `XRPTestUtils` too? (Or you can just do it in this PR if you're feeling excited about it).